BookUniverse.BLL/Services/AuthenticationService.cs

```diff
@@ -62,7 +62,22 @@ public async Task<User> Login(LoginDto user)
             return storedAccount;
         }
 
-        public bool IsLoggedIn()
+        public async Task EditUser(EditUserDto new_user)
```

Comment:
maybe it will be better to move this method to UserService?

Reply:
Also if you add new method to service implementation, you need to declare this method in interface